code/API_definitions/BYON-CallHandling-Service.yaml

```diff
@@ -460,3 +523,14 @@ components:
             status: 503
             code: UNAVAILABLE
             message: Service unavailable
+    Generic201:
+      description: Created
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ReceiverSessionStatus'
+          examples:
```

Comment:
These examples are not relevant here.

Reply:
201 Created is to convey to client that the request is accepted, and resource is created on WebRTC Gateway.  After 201, the gateway shall initiate the SIP transactions towards the network, based on transactions, subsequent notification shall be sent. 
My point was, listing exMT201_183 and exMT201_200 to be removed.